src/api/payment-requests/payment-requests.md

```diff
@@ -467,12 +467,168 @@ them to find the Payment Request and proceed to pay.
 {% endjson %}
 
 
+### Pay a Payment Request by Id
+
+There are two methods of paying a payment request.
+The first uses Centrapay [Assets] and requires you to provide the Id of the asset to use for payment.
+Alternatively you can provide a transaction hash and the Centrapay [Asset Type] for any crypto payments that we support.
+
+{% reqspec %}
+  POST '/api/payment-requests/{paymentRequestId}/payment'
+  auth 'jwt'
+  example {
+    title 'Pay a Payment Request with a Centrapay asset'
+    body ({
+      "assetId": "WRhAxxWpTKb5U7pXyxQjjY"
+    })
+  }
+
+  example {
+    title 'Pay a Payment Request using transaction hash from a crypto payment'
+    body ({
+      "assetType": "bitcoin.main",
+      "transactionHash": "VMXMkUttDGTVz4ESv5ND56",
+    })
+  }
+
+{% endreqspec %}
+
+{% h4 Example response payload %}
+
+{% json %}
+{
+  "id": "MhocUmpxxmgdHjr7DgKoKw",
+  "url": "https://app.centrapay.com/pay/MhocUmpxxmgdHjr7DgKoKw",
+  "merchantId": "26d3Cp3rJmbMHnuNJmks2N",
+  "merchantName": "Centrapay Café",
+  "configId": "5efbe2fb96c08357bb2b9242",
+  "value": {
+    "currency": "NZD",
+    "amount": "1000"
+  },
+  "paymentOptions": [
+    {
+      "amount": "1000",
+      "assetType": "centrapay.nzd.main"
+    },
+    {
+      "amount": "1000",
+      "assetType": "cca.coke.main"
+    }
+  ],
+  "status": "paid",
+  "createdAt": "2021-06-08T04:04:27.426Z",
+  "updatedAt": "2021-06-08T04:04:27.426Z",
+  "expiresAt": "2021-06-08T04:06:27.426Z",
+  "liveness": "main",
+  "expirySeconds": 120,
+  "paidBy": {
+    "assetTotals": [
+      {
+        "type": "centrapay.nzd.main",
+        "description": "Centrapay NZD",
+        "settlementDate": "2021-06-28T04:04:27.426Z",
+        "total": {
+          "amount": "550",
+          "currency": "NZD"
+        }
+      }
+    ]
+  }
+}
+{% endjson %}
+
+### Refund a Payment Request by Id
+
+{% reqspec %}
+  POST '/api/payment-requests/{paymentRequestId}/refund'
+  auth 'jwt'
+  example {
+    title 'Refund a Payment Request'
+    body ({
+      "value": {
+        "amount": "100",
+        "currency": "NZD",
+      },
+      "externalReference": "2021-12-01T01:31:46+00:00",
```

Comment:
This guy surprised me - I had no idea what it meant. A couple of questions:
- The [legacy docs](https://docs.centrapay.com/api/legacy-payment-requests#refunding-a-transaction) describe what a refund with an external reference is. Do we want a similar section here?
- What about the option to refund without an external reference? Have we considered supporting that / do we need to?

Reply:
Sorry, it is an optional field, we should add documentation that explains that. The original need for the external reference in order to do partial refunds isn't going to be needed anymore.